stories/shot-card.stories.js

```diff
@@ -1,138 +0,0 @@
-import React from 'react';
-import { stories } from './card.stories';
```

Comment:
This was a weird case where we imported the Card stories to add additional stories in this file.  This doesn't really work in Component Story Format, so these stories were folded into `card.stories.js`.